yoti_python_sdk/activity_details.py

```diff
@@ -48,53 +46,37 @@ def try_parse_selfie_field(self, field):
             field.content_type
         )
 
-    def try_parse_age_verified_field(self, field, anchors):
+    def try_parse_age_verified_field(self, field):
         if field is not None:
             is_age_verified = Protobuf().value_based_on_content_type(
                 field.value,
                 field.content_type
             )
             if is_age_verified == 'true':
```

Comment:
This currently catches the (very unlikely) case that `is_age_verified` is a value other than `true` or `false` - as we would want to throw an error in that case.  Although removing it would make it more concise...

Reply:
@bucky-boy I think it would be be better to leave it, to be safe